src/test/cypress/support/pageobjects/exam/ExamParticipation.ts

```diff
@@ -132,7 +132,9 @@ export class ExamParticipation {
     }
 
     verifyExerciseTitleOnFinalPage(exerciseID: number, exerciseTitle: string) {
-        getExercise(exerciseID).find(`#exercise-group-title-${exerciseID}`).contains(exerciseTitle).should('be.visible');
+        const titleSelector = getExercise(exerciseID).find(`#exercise-group-title-${exerciseID}`);
+        titleSelector.scrollIntoView();
+        titleSelector.contains(exerciseTitle).should('be.visible');
```

Comment:
Please don't save command results in variables, see https://docs.cypress.io/guides/references/best-practices#Assigning-Return-Values.

Reply:
I read the guide the way that you shouldn't use it around state changes. I, however use it only to not query the same field twice. Chaining scrollIntoView is not recommended.

How would you implement it? Defining an alias here and technically redefining it whenever calling the method seems excessive. And I don't intend to do a large refactoring in this PR.

Reply:
I was thinking of 
```
getExercise().find().as().scrollIntoView();
get().contains().should();
```

Reply:
I mean I did it deliberately because I didn't want to query it multiple times. But in the end - cypress will be deprecated soon anyway now that playwright got merged so it doesn't matter too much anyway